frontend/src/components/Robots/RobotAvatar/index.tsx

```diff
@@ -4,7 +4,7 @@ import { Avatar, Badge, Tooltip } from '@mui/material';
 import { useTranslation } from 'react-i18next';
 import { SendReceiveIcon } from '../../Icons';
 
-interface DepthChartProps {
+interface Props {
```

Comment:
It's good to have always identified the props per component, but I don't have a strong opinion neither 👍

Reply:
I see, sounds good to me. I realized we are not being consistent across components. It was easier to change this one rather than all the other ones :smile: